.changeset/auth-token-command.md

```diff
@@ -0,0 +1,13 @@
+---
+"wrangler": minor
+---
+
+Add `wrangler auth token` command to retrieve the current OAuth token, refreshing it if necessary.
```

Comment:
IIUC this is not necesarily a OAuth token (as described in the PR description) - it could be an API token (created via the dashboard)